src/main/java/picard/util/AlleleSubsettingUtils.java

```diff
@@ -0,0 +1,267 @@
+package picard.util;
+
+import htsjdk.variant.variantcontext.*;
+import picard.fingerprint.Snp;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+/**
+ * Utilities class containing methods for restricting {@link VariantContext} and {@link GenotypesContext} objects to a
+ * reduced set of alleles, as well as for choosing the best set of alleles to keep and for cleaning up annotations and
+ * genotypes after subsetting.
+ *
+ * @author David Benjamin davidben@broadinstitute.org;
+ * @author Yossi Farjoun farjoun@broadinstitute.org;
+ */
+
+public final class AlleleSubsettingUtils {
+    private AlleleSubsettingUtils() {
+    }  // prevent instantiation
+
+    @Deprecated // use {@link Allele.NON_REF} (once released and htsjdk is properly reved in picard)
```

Comment:
Is this going to be released with this deprecation in place? If so, the use comment should be put in a javadoc.

Reply:
done.